packages/core/src/components/AutocompleteField/AutocompleteField.scss

```diff
@@ -0,0 +1,108 @@
+// Form-specific variables can be found in _variables.forms
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Autocomplete field
+
+@paypal https://github.com/paypal/downshift#itemtostring
+
+Autocomplete fields allow users to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted), and receive one or more suggested matches in a list below the input.
+
+disabled - Disabled autocomplete
+
+Markup:
+<div class="ds-u-display--flex ds-u-flex-wrap--wrap ds-c-autocomplete">
+  <label class="ds-c-label ds-u-margin-top--0" for="autocomplete_7">
+    <span class="ds-c-autocomplete__label">What zip code did this person live during 2017?</span>
+    <span class="ds-c-field__hint">This is an autocomplete field. Begin typing to search for relevant information. The number of results will be updated as you type.</span>
+  </label>
+  <input aria-autocomplete="list" aria-expanded="false" autocomplete="off" class="ds-c-field ds-c-autocomplete__input" id="autocomplete_7" role="combobox" value="" {{modifier}}>
+  <a aria-label="Clear typeahead and search again" class="ds-c-button ds-c-button--transparent ds-c-button--small ds-u-margin-left--auto ds-u-padding-right--0 ds-c-autocomplete__button" href="javascript:void(0);">Search again</a>
+</div>
+
+Style guide: components.autocomplete-field
+*/
+
+// autocomplete component
+.ds-c-autocomplete {
+  max-width: 460px;
+
+  &__label {
+    + .ds-c-field__hint {
+      border: 0;
+      clip: rect(0, 0, 0, 0);
+      height: 1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+      word-wrap: normal;
+    }
+  }
+
+  &__input[disabled] {
+    + .ds-c-autocomplete__button {
+      background: transparent;
+      border: transparent;
+      color: $color-gray-dark;
+      pointer-events: none;
+    }
+  }
+
+  &__list {
```

Comment:
Should this list be absolute positioned so it doesn't push other page content down as the user is typing? 

![autocomplete](https://user-images.githubusercontent.com/371943/33838078-63029ee6-de5c-11e7-89e8-89964876a933.gif)

Reply:
Yes, using position: absolute is a good call. This will be included in the next PR.